voting_schemes/electionguard/python-wrapper/src/bulletin_board/electionguard/serializable.py

```diff
@@ -59,27 +60,27 @@ def maybe_base64_to_int(i: Union[str, int]) -> int:
 
 
 def set_serializers():
-    old_set_serializers()
+    # old_set_serializers()
```

Comment:
⚠️ should we keep this? if not, we should delete this commented code and the assignment of this function

Reply:
Kept it!